web_app/templates/home.html

```diff
@@ -3,16 +3,6 @@
 
 {% block content %}
 
-    <!DOCTYPE html>
-    <html lang="en">
```

Comment:
since the templates inherit from the shared layout, and the shared layout has the complete HTML document markup, we don't need that markup again within the individual pages. just a content block to inject into the layout.